[PATCH] export linux/a.out.h

Export linux/a.out.h like we do for asm/a.out.h. One minor complication is
that the interesting stuff is protected by an CONFIG_ARCH_SUPPORTS_AOUT ifdef,
so make sure this gets defined for unifdef.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 include/linux/Kbuild |    2 ++
 1 file changed, 2 insertions(+)

Index: linux-2.6.25.6/include/linux/Kbuild
===================================================================
--- linux-2.6.25.6.orig/include/linux/Kbuild
+++ linux-2.6.25.6/include/linux/Kbuild
@@ -159,6 +159,8 @@
 header-y += videotext.h
 header-y += x25.h
 
+UNIFDEF += -DCONFIG_ARCH_SUPPORTS_AOUT
+unifdef-y += a.out.h
 unifdef-y += acct.h
 unifdef-y += adb.h
 unifdef-y += adfs_fs.h
